include/particles/particle.tcc

```diff
@@ -238,10 +238,10 @@ void mpm::Particle<Tdim>::initialise() {
   volumetric_strain_centroid_ = 0.;
 
   // Initialize vector data properties
-  this->properties_["stresses"] = [&]() { return stress(); };
-  this->properties_["strains"] = [&]() { return strain(); };
-  this->properties_["velocities"] = [&]() { return velocity(); };
-  this->properties_["displacements"] = [&]() { return displacement(); };
+  properties_["stresses"] = [&]() { return stress(); };
+  properties_["strains"] = [&]() { return strain(); };
+  properties_["velocities"] = [&]() { return velocity(); };
+  properties_["displacements"] = [&]() { return displacement(); };
```

Comment:
Are you planning to combine this with the new `vector_properties`?

Reply:
We'll remove it when we have vector properties fully integrated.